### ISSUE TITLE
Use a static site generator

### ISSUE THREAD

**Issue by thangngoc89** [thangngoc89/openit.io#5]
Currently find and replace multiple files at once, in proxy/templates.

Best option should be translating README to Html and embed it at build time.

Any suggestions? Hugo? Metalsmith? Self-build?

**Comment by strugee** [comment]
You don't need much for this since you don't have a blog or anything. I'd suggest not using Hugo or Metalsmith since that introduces a lot of complexity that you don't need; you'll probably be okay just wiring up Pug or something using [gulp](http://gulpjs.com/).

This is my opinion obviously - feel free to ignore ;)

**Comment by thangngoc89** [comment]
@strugee Yeah. Thank you for the suggestion. I wrote a dead simple script to convert markdown in README to index.html with node.js. Gonna push it to this repository soon. Using a full-blown static site generator is overengineer

**Comment by strugee** [comment]
:thumbsup: sweet!
